cranelift/filetests/filetests/isa/x86/bnot-b1.clif

```diff
@@ -0,0 +1,13 @@
+test binemit
+test run
+
+target x86_64
+
+function u0:323() -> b1 system_v {
+block0:
+    [-,%rax] v221 = bconst.b1 false   ; bin: 40 b8 00000000
+    [-,%rcx] v222 = bconst.b1 false   ; bin: 40 b9 00000000
+    [-,%rax] v223 = bnot v221         ; bin: 40 f7 d0
+    [-,%rax] v224 = band v223, v222   ; bin: 40 21 c8
+    return v224
+}
```

Comment:
To actually run this test, this needs a `; run` after the function. Then it should fail, since the logic of `band true, false` should be false and the test runner will interpret that as a failed test. Perhaps change `v222` to `true`?

Reply:
> @abrown Your review doesn't contain any code comments.

@bjorn3, that's because I'm slow...

Reply:
Done